src/lib.rs

```diff
@@ -37,7 +37,11 @@
 //! - `use_std`
 //!   - Enabled by default.
 //!   - Disable to compile itertools using `#![no_std]`. This disables
-//!     any items that depend on collections (like `chunk_by`, `unique`,
+//!     any item that depend on allocations (see the `use_alloc` feature)
+//!     and hashes (like `unique`, `counts`, `into_grouping_map` and more).
```

Comment:
```suggestion
//!     and hash maps (like `unique`, `counts`, `into_grouping_map` and more).
```

Reply:
There is also "hash sets" so I chose "hashes" to unify those two terms.